.github/workflows/check-oss-history.yml

```diff
@@ -0,0 +1,31 @@
+name: Check OSS history
+on: pull_request_target
+
+jobs:
+  contribs:
+    runs-on: ubuntu-latest
+    name: Check OSS history
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+        with:
+          path: ncs/nrf
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: Install extra python dependencies
+        run: |
+          pip3 install west
+          pip3 install -r ncs/nrf/scripts/requirements-extra.txt
+
+      - name: Set up the workspace
+        run: |
+          west init -l ncs/nrf
+          cd ncs
+          west update zephyr
+          git -C zephyr remote add upstream https://github.com/zephyrproject-rtos/zephyr
+
+      - name: Check OSS history
+        uses: mbolivar-nordic/action-check-oss-history@main
```

Comment:
@mbolivar-nordic Can you give me admin access to this repo so I can move it to the nrfconnect org?

Reply:
> @mbolivar-nordic Can you give me admin access to this repo so I can move it to the nrfconnect org?

I sent you an invite to join as a collaborator but I don't see a way to add you as an admin. Why not just make a new repo and push the current contents in btw?